implementations/rust/ockam/ockam_command/tests/bats/portals.bats

```diff
@@ -40,7 +40,7 @@ teardown() {
 
   # Test deletion of a previously deleted TCP inlet
   run_failure $OCKAM tcp-inlet delete "test-inlet" --at /node/n2 --yes
-  assert_output --partial "NotFound"
+  assert_output --partial "not found"
```

Comment:
The full message is now more explicit but the `OCKAM` error code is `500` (before and after the PR) where it should rather be `404`